src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -7,190 +7,190 @@ partial class ApplicationEngine
     {
         public static readonly IReadOnlyDictionary<OpCode, long> OpCodePrices = new Dictionary<OpCode, long>
         {
-            [OpCode.PUSHINT8] = 30,
-            [OpCode.PUSHINT16] = 30,
-            [OpCode.PUSHINT32] = 30,
-            [OpCode.PUSHINT64] = 30,
-            [OpCode.PUSHINT128] = 120,
-            [OpCode.PUSHINT256] = 120,
-            [OpCode.PUSHA] = 120,
-            [OpCode.PUSHNULL] = 30,
-            [OpCode.PUSHDATA1] = 180,
-            [OpCode.PUSHDATA2] = 13000,
-            [OpCode.PUSHDATA4] = 110000,
-            [OpCode.PUSHM1] = 30,
-            [OpCode.PUSH0] = 30,
-            [OpCode.PUSH1] = 30,
-            [OpCode.PUSH2] = 30,
-            [OpCode.PUSH3] = 30,
-            [OpCode.PUSH4] = 30,
-            [OpCode.PUSH5] = 30,
-            [OpCode.PUSH6] = 30,
-            [OpCode.PUSH7] = 30,
-            [OpCode.PUSH8] = 30,
-            [OpCode.PUSH9] = 30,
-            [OpCode.PUSH10] = 30,
-            [OpCode.PUSH11] = 30,
-            [OpCode.PUSH12] = 30,
-            [OpCode.PUSH13] = 30,
-            [OpCode.PUSH14] = 30,
-            [OpCode.PUSH15] = 30,
-            [OpCode.PUSH16] = 30,
-            [OpCode.NOP] = 30,
-            [OpCode.JMP] = 70,
-            [OpCode.JMP_L] = 70,
-            [OpCode.JMPIF] = 70,
-            [OpCode.JMPIF_L] = 70,
-            [OpCode.JMPIFNOT] = 70,
-            [OpCode.JMPIFNOT_L] = 70,
-            [OpCode.JMPEQ] = 70,
-            [OpCode.JMPEQ_L] = 70,
-            [OpCode.JMPNE] = 70,
-            [OpCode.JMPNE_L] = 70,
-            [OpCode.JMPGT] = 70,
-            [OpCode.JMPGT_L] = 70,
-            [OpCode.JMPGE] = 70,
-            [OpCode.JMPGE_L] = 70,
-            [OpCode.JMPLT] = 70,
-            [OpCode.JMPLT_L] = 70,
-            [OpCode.JMPLE] = 70,
-            [OpCode.JMPLE_L] = 70,
-            [OpCode.CALL] = 22000,
-            [OpCode.CALL_L] = 22000,
-            [OpCode.CALLA] = 22000,
-            [OpCode.ABORT] = 30,
-            [OpCode.ASSERT] = 30,
-            [OpCode.THROW] = 22000,
-            [OpCode.TRY] = 100,
-            [OpCode.TRY_L] = 100,
-            [OpCode.ENDTRY] = 100,
-            [OpCode.ENDTRY_L] = 100,
-            [OpCode.ENDFINALLY] = 100,
+            [OpCode.PUSHINT8] = 1 << 0,
+            [OpCode.PUSHINT16] = 1 << 0,
+            [OpCode.PUSHINT32] = 1 << 0,
+            [OpCode.PUSHINT64] = 1 << 0,
```

Comment:
These PUSHes (as well as PUSHM1..PUSH16) do cost more than NOP, I'd suggest `1<<2` here based on data we have (and it's very similar for two implementations with real costs around 4-5 NOPs).